.github/workflows/backend.yml

```diff
@@ -92,6 +92,7 @@ jobs:
         uses: ./.github/actions/setup-sentry
         id: setup
         with:
+          redis_cluster: true
```

Comment:
I think this makes sense, but we also have deployments that don't use redis cluster. How will we test the alternate configuration?

Reply:
both services are running, but by default they are configured to use `redis` unless explicitly overridden

Reply:
I see, so we need to explicitly use 
```
@override_settings(SENTRY_PROCESSING_SERVICES={"redis": {"redis": "cluster"}})
@override_options(get_redis_cluster_default_options(id="cluster"))
```

To test cluster in specific places

Reply:
yes 👍